src/main/java/Duke.java

```diff
@@ -1,10 +1,157 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final int MAXTASKSIZE = 100;
```

Comment:
Good job using uppercase for constants, but you can also use underscores to show spaces